pytest_tests/lib/helpers/complex_object_actions.py

```diff
@@ -275,17 +275,15 @@ def get_nodes_with_object(
 
     nodes_list = []
     for node in nodes:
-        wallet = node.wallet.path
-        wallet_config = neofs_env.generate_cli_config(node.wallet)
```

Comment:
Unnecessary change.

Reply:
Moved it to a separate commit, otherwise we are regenerating cli config every time this function is called.